yt_dlp/extractor/biqle.py

```diff
@@ -32,53 +36,70 @@ class BIQLEIE(InfoExtractor):
             'id': '-44781847_168547604',
             'ext': 'mp4',
             'title': 'Ребенок в шоке от автоматической мойки',
+            'description': 'Ребенок в шоке от автоматической мойки — Смотреть онлайн',
             'timestamp': 1396633454,
-            'uploader': 'Dmitry Kotov',
             'upload_date': '20140404',
-            'uploader_id': '47850140',
+            'thumbnail': r're:https://[^/]+/c535507/u190034692/video/l_b84df002\.jpg',
         },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
-        embed_url = self._proto_relative_url(self._search_regex(
-            r'<iframe.+?src="((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^"]+)".*?></iframe>',
-            webpage, 'embed url'))
+
+        title = self._search_regex(
+            r'<meta.*itemprop ?= ?"name".*content ?= ?"([^"]+)".*/>',
```

Comment:
This is how the regex should be if _html_search_meta wasnt available

```suggestion
            r'<meta[^>]+itemprop\s*=\s*"name"[^>]+content\s*=\s*"([^">]+)"',
```